### PR TITLE
Update readme and hardware access links

### PR DESCRIPTION
**Pull request by josh146**
* Updated readme can be viewed [here](https://github.com/XanaduAI/strawberryfields/tree/update-hardware-access-links)

* Updated docs landing page can be viewed [here](https://strawberryfields--448.org.readthedocs.build/en/448/)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -39,16 +39,22 @@ Features
 
     <img src="https://raw.githubusercontent.com/XanaduAI/strawberryfields/master/doc/_static/StateLearning.gif" width="300px"  align="right">
 
-* An open-source software architecture for **photonic quantum computing**
+* Execute photonic quantum algorithms directly on `Xanadu's next-generation quantum hardware.
```
Comment (co9olguy) [inline README.rst:42]:
```suggestion
* Execute photonic quantum algorithms directly on `Xanadu's next-generation quantum hardware
```

File: README.rst [hunk 2]
```diff
@@ -39,16 +39,22 @@ Features
...
+  <https://strawberryfields.ai/photonics/hardware/index.html>`__
 
-* Execute algorithms on  **photonic quantum computing chips** via the Xanadu Quantum Cloud
+* `High-level functions <https://strawberryfields.ai/photonics/applications.html>`__ for **solving
+  practical problems** including **graph and network optimization**, **machine learning** and
+  **chemistry**
 
-* Includes high-level functions for **solving practical problems** including **graph and network
-  optimization**, **machine learning** and **chemistry**
+* Includes a suite of world-class simulators—based on cutting edge algorithms—to compile and
```
Comment (co9olguy) [inline README.rst:49]:
```suggestion
* Includes a suite of world-class simulators—based on cutting-edge algorithms—to compile and
```

File: README.rst [hunk 3]
```diff
@@ -39,16 +39,22 @@ Features
...
+  simulate photonic algorithms.
```
Comment (co9olguy) [inline README.rst:50]:
```suggestion
  simulate photonic algorithms
```

File: README.rst [hunk 4]
```diff
@@ -39,16 +39,22 @@ Features
...
 
-* Includes **quantum simulators** implemented using **NumPy** and **TensorFlow**—with built-in quantum compilation
+* `Train and optimize
+  <https://strawberryfields.ai/photonics/demonstrations.html#optimization-and-machine-learning>`__
+  your quantum programs with our end-to-end differentiable TensorFlow backend.
```
Comment (co9olguy) [inline README.rst:54]:
```suggestion
  your quantum programs with our end-to-end differentiable TensorFlow backend
```

File: README.rst [hunk 5]
```diff
@@ -39,16 +39,22 @@ Features
...
+  **chemistry**
 
-* Includes high-level functions for **solving practical problems** including **graph and network
-  optimization**, **machine learning** and **chemistry**
+* Includes a suite of world-class simulators—based on cutting edge algorithms—to compile and
+  simulate photonic algorithms.
 
-* Includes **quantum simulators** implemented using **NumPy** and **TensorFlow**—with built-in quantum compilation
+* `Train and optimize
+  <https://strawberryfields.ai/photonics/demonstrations.html#optimization-and-machine-learning>`__
```
Comment (co9olguy) [inline README.rst:53]: Title cut off when linked page is loaded :(

File: README.rst [hunk 6]
```diff
@@ -64,14 +70,29 @@ Strawberry Fields requires Python version 3.6, 3.7, or 3.8. Installation of Stra
 Getting started
 ===============
 
-To see Strawberry Fields in action immediately, try out our `Strawberry Fields Interactive <https://strawberryfields.ai>`_ web application. Prepare your initial states, drag and drop gates, and watch your simulation run in real time right in your web browser.
+To see Strawberry Fields in action immediately, try out our `Strawberry Fields Interactive
+<https://strawberryfields.ai>`_ web application. Prepare your initial states, drag and drop gates,
+and watch your simulation run in real time right in your web browser.
```
Comment (co9olguy) [inline README.rst:75]: Are you going to keep the progress bar? :)

File: README.rst [hunk 7]
```diff
@@ -64,14 +70,29 @@ Strawberry Fields requires Python version 3.6, 3.7, or 3.8. Installation of Stra
...
 
-For getting started with writing your own Strawberry Fields code, follow the `installation <https://strawberryfields.readthedocs.io/en/stable/_static/install.html>`_ page to get Strawberry Fields up and running, then jump over to the `tutorials <https://strawberryfields.readthedocs.io/en/stable/introduction/tutorials.html>`_ to see what you can do.
+For getting started with writing your own Strawberry Fields code, begin with our `photonic circuit
```
Comment (co9olguy) [inline README.rst:77]:
```suggestion
To get started with writing your own Strawberry Fields code, begin with our `photonic circuit
```

File: doc/index.rst [hunk 8]
```diff
@@ -95,23 +95,32 @@ Features
     :width: 300px
     :target: javascript:void(0);
 
-* An open-source software architecture for **photonic quantum computing**
+
+* Execute photonic quantum algorithms directly on `Xanadu's next-generation quantum hardware.
```
Comment (co9olguy) [inline doc/index.rst:99]:
```suggestion
* Execute photonic quantum algorithms directly on `Xanadu's next-generation quantum hardware
```

File: doc/index.rst [hunk 9]
```diff
@@ -95,23 +95,32 @@ Features
...
+  <https://strawberryfields.ai/photonics/hardware/index.html>`__
 
 ..
 
-* Includes high-level functions for **solving practical problems** including **graph and network optimization**, **machine learning** and **chemistry**.
+* `High-level functions <https://strawberryfields.ai/photonics/applications.html>`__ for **solving
+  practical problems** including **graph and network optimization**, **machine learning** and
```
Comment (co9olguy) [inline doc/index.rst:105]:
```suggestion
  practical problems** including **graph and network optimization**, **machine learning**, and
```

File: README.rst [hunk 10]
```diff
@@ -39,16 +39,22 @@ Features
 
     <img src="https://raw.githubusercontent.com/XanaduAI/strawberryfields/master/doc/_static/StateLearning.gif" width="300px"  align="right">
 
-* An open-source software architecture for **photonic quantum computing**
+* Execute photonic quantum algorithms directly on `Xanadu's next-generation quantum hardware.
+  <https://strawberryfields.ai/photonics/hardware/index.html>`__
 
-* Execute algorithms on  **photonic quantum computing chips** via the Xanadu Quantum Cloud
+* `High-level functions <https://strawberryfields.ai/photonics/applications.html>`__ for **solving
+  practical problems** including **graph and network optimization**, **machine learning** and
```
Comment (co9olguy) [inline README.rst:46]:
```suggestion
  practical problems** including **graph and network optimization**, **machine learning**, and
```

File: doc/index.rst [hunk 11]
```diff
@@ -95,23 +95,32 @@ Features
...
+  **chemistry**
 
 ..
 
-* Includes **quantum simulators** implemented using **NumPy** and **TensorFlow**—with built-in quantum compilation
+* Includes a suite of world-class simulators—based on cutting edge algorithms—to compile and
+  simulate photonic algorithms.
```
Comment (co9olguy) [inline doc/index.rst:111]:
```suggestion
  simulate photonic algorithms
```

File: doc/index.rst [hunk 12]
```diff
@@ -95,23 +95,32 @@ Features
...
 
 ..
 
-* Future releases will target experimental backends, including **photonic quantum computing chips**
+* `Train and optimize
+  <https://strawberryfields.ai/photonics/demonstrations.html#optimization-and-machine-learning>`__
+  your quantum programs with our end-to-end differentiable TensorFlow backend.
```
Comment (co9olguy) [inline doc/index.rst:117]:
```suggestion
  your quantum programs with our end-to-end differentiable TensorFlow backend
```

**Review by co9olguy — CHANGES_REQUESTED** [review]

File: README.rst (hunk 5 above)
Comment (josh146) [reply README.rst:53]: :(

File: README.rst (hunk 6 above)
Comment (josh146) [reply README.rst:75]: For now!

**Review by co9olguy — APPROVED** [review]

**Pull request merged**